bitnami/rabbitmq/Chart.yaml

```
@@ -23,4 +23,4 @@ name: rabbitmq
 sources:
   - https://github.com/bitnami/bitnami-docker-rabbitmq
   - https://www.rabbitmq.com
-version: 8.25.1
+version: 9.0.0
```

```suggestion
version: 8.26.0
```

Let's do a minor instead.

---

Refactored it into a minor release.